gluten-data/src/main/java/io/glutenproject/exec/ExecutionCtxJniWrapper.java

```diff
@@ -20,7 +20,7 @@ public class ExecutionCtxJniWrapper {
 
   private ExecutionCtxJniWrapper() {}
 
-  public static native long createExecutionCtx();
+  public static native long createExecutionCtx(byte[] sessionConf);
```

Comment:
Just as a TODO: We'd avoid relying on substrait's Protobuf definition for transmitting configs. We had added Gluten's own proto container `gluten/gluten-core/src/main/resources/io/glutenproject/proto`